src/controllers/aliases/AliasesName.hpp

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include "controllers/accounts/AccountController.hpp"
+
+#include "util/RapidJsonSerializeQString.hpp"
+#include "util/RapidjsonHelpers.hpp"
+
+#include <QString>
+#include <pajlada/serialize.hpp>
+
+#include <memory>
+
+namespace chatterino {
+
+class AliasesName
+{
+public:
+    AliasesName(const QString &name, const QString &replace)
+        : name_(name)
+        , replace_(replace)
+    {
+    }
+
+    const QString &getName() const
```

Comment:
Other [getters](https://wiki.chatterino.com/Contributing%20for%20Developers/#members) in the app follow a pattern like
```suggestion
    const QString &name() const
```

Reply:
I think I copied the code from https://github.com/Chatterino/chatterino2/blob/master/src/controllers/filters/FilterRecord.hpp#L43 but I guess it doesn't really matter.